vignettes/Spectra.Rmd

```diff
@@ -415,9 +425,48 @@ peaks data (m/z and intensity values) within each spectrum are:
 - `reduceSpectra`: filters individual spectra keeping only the largest peak for
   groups of peaks with similar m/z values.
 
-In the example below we select all spectra measured in the second mzML file and
-subsequently filter them to retain spectra measured between 175 and 189 seconds
-in the measurement run.
+In this example, we use the filterValues function to retain spectra with a base
+peak m/z close to 100 (+/- 30 ppm) and a retention time around 230 (+/- 5 s).
+
+```{r}
+sps_sub <- filterValues(sps_sciex, spectraVariables = c("basePeakMZ", "rtime"),
+                        values = c(123.089,230), tolerance = c(0,5), 
+                        ppm = c(30, 0), condition = "all")
+length(sps_sub)
+```
+
+Then, we demonstrate the usage of the `filterRanges` function to filter spectra
+based on ranges of values for variables such as base peak m/z, peak count, and
+retention time. 
+
+```{r}
+sps_ranges <- filterRanges(sps_sciex,
+                           spectraVariables = c("basePeakMZ","peaksCount",
+                                                "rtime"),
+                           ranges = c(123.09,124, 3500, 3520, 259, 260),
+                           condition = "all")
+length(sps_ranges)
+```
+
+Only one spectrum matches all the ranges. Another option for filterValues and
```

Comment:
please always add backticks around actual function (or object) names as well as for R code, such as `condition = "any"`.